x-pack/plugins/security_solution/public/plugin_contract.ts

```diff
@@ -21,66 +19,64 @@ export class PluginContract {
   public componentsService: ContractComponentsService;
   public upsellingService: UpsellingService;
   public onboardingPageService: OnboardingPageService;
-  public extraRoutes$: BehaviorSubject<RouteProps[]>;
-  public appLinksSwitcher: AppLinksSwitcher;
-  public deepLinksFormatter?: DeepLinksFormatter;
```

Comment:
Not needed anymore in the contract, now they are implemented inside the _security_solution_ plugin